nixos/modules/services/desktop-managers/plasma6.nix

```diff
@@ -87,30 +88,27 @@ in {
         # Core Plasma parts
         kwin
         pkgs.xwayland
-
         kscreen
         libkscreen
-
         kscreenlocker
-
         kactivitymanagerd
         kde-cli-tools
-        kglobalacceld
+        kglobalacceld # keyboard shortcut daemon
         kwrited # wall message proxy, not to be confused with kwrite
-
-        milou
-        polkit-kde-agent-1
-
+        baloo # system indexer
+        milou # search engine atop baloo
+        ffmpegthumbs # video thumbnailer
+        kdegraphics-thumbnailers # pdf etc thumbnailer
+        polkit-kde-agent-1 # polkit auth ui
         plasma-desktop
         plasma-workspace
-
-        # Crash handler
-        drkonqi
+        drkonqi # crash handler
+        kde-inotify-survey # handles running out of inodes
```

Comment:
That's not what this does

Reply:
kde-inotify-survey? It warns the user when they run out of inodes, no?

Reply:
No, it warns the user about low inotify watch limits.